### ISSUE TITLE
Android - Blank space displayed under compose box on new conversations

### ISSUE THREAD

**Issue by isagoico** [Expensify/App#10539]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:
1. Log in the Android app
2. Navigate to a new conversation
3. Tap on the compose box to start writing

## Expected Result:
Compose box is displayed on top of the keyboard without any blank spaces. 

## Actual Result:
There's a blank space under the compose box that is pushing the compose box to the top. 

_Note:_ this is not happening with conversations that have existing messages.

## Workaround:
User can still type on the compose box but it unable to view the conversation area. 

## Platform:

Where is this issue occurring?

- Android

**Version Number:** 1.1.1-89

**Reproducible in staging?:** Yes
**Reproducible in production?:** No

**Email or phone of affected tester (no customers):** isagoicotest@gmail.com / 123456Isa

**Logs:** https://stackoverflow.com/c/expensify/questions/4856

**Notes/Photos/Videos:** Any additional supporting documentation

<img src="https://user-images.githubusercontent.com/44479856/186478377-facd655c-5119-4fd8-b5fa-f5396b962b9b.png" width="40%">

**Expensify/Expensify Issue URL:** 

**Issue reported by:** Applause
**Slack conversation:** N/A

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by isagoico** [comment]
As mentioned above, unable to reproduce the issue in production. Below is the behavior in prod app:

<img src="https://user-images.githubusercontent.com/44479856/186479095-8cf9b5ac-c93a-4b25-bc37-ce83428a0059.png" width="40%">

**Comment by sketchydroide** [comment]
@isagoico do you have a video you can add? thanks it's easier to replicate/understand when there is a video

**Comment by isagoico** [comment]
Yep! Adding it now - Looks like the "beginning of your chat" is not being rendered in the conversation 🤔 maybe this is happening because of that. 

Attaching the video here and to the OC. 

https://user-images.githubusercontent.com/44479856/186482134-e8d906db-078c-4574-8795-d9320771f5e7.mp4

**Comment by roryabraham** [comment]
Looking into this

**Comment by roryabraham** [comment]
The problem _appears_ to be that the `ReportActionsList` isn't rendering at all:

<img width="268" alt="image" src="https://user-images.githubusercontent.com/47436092/186507654-cfbdb29b-fea8-4ca1-bb43-2a21dedb8abb.png">

And I saw this error which might be a hint:

<img width="907" alt="image" src="https://user-images.githubusercontent.com/47436092/186507788-f1062048-4852-4a76-a490-ea0aa18b9fc0.png">

**Comment by roryabraham** [comment]
[Diagnosed the problem](https://expensify.slack.com/archives/C03UW4UU1C5/p1661372193647169?thread_ts=1661360904.807919&cid=C03UW4UU1C5), but don't have a fix ready

**Comment by dhairyasenjaliya** [comment]
looks like this issue is the similar issue or might be the same which is  logged here https://github.com/Expensify/App/issues/10441

**Comment by Julesssss** [comment]
This occurred for me the very first time I opened a new chat, but it's inconsistent.

**Comment by tgolen** [comment]
Trying to get a C+ review of https://github.com/Expensify/App/pull/10648 not sure if I did it right 😓

**Comment by Julesssss** [comment]
Hmm, I'm trying to get a C+ team member to review this improved [fix PR](https://github.com/Expensify/App/pull/10648). Does anyone know why this assigned a Contributor manager instead?

**Comment by Julesssss** [comment]
Hey @thesahindia 👋 

Please review my PR here, which reverts a RN bugfix and replaces it with a native fix. Ideally, you would test on a physical Android device. But please let me know if you're not able to do this!

**Comment by thesahindia** [comment]
Hey @Julesssss, it will take me a few hours. I had to reset my mac, it was acting funny😓

**Comment by michaelhaxhiu** [comment]
Should this be open till https://github.com/Expensify/App/pull/10648 is merged to prod?
